src/main.js

```diff
@@ -301,6 +313,20 @@ function TimekitBooking() {
 
   };
 
+  // Display ribbon if in testmode
+  var renderTestRibbon = function() {
```

Comment:
Perhaps rename to `renderTestModeRibbon` just to avoid potential confusion. `test ribbon` could sound like something related to debugging or unit tests.

Reply:
sure, I had actually recently changed it away from test mode haha :D
but i see your point.